stable/mongodb-replicaset/values.yaml

```diff
@@ -11,11 +11,14 @@ auth:
   enabled: false
   # adminUser: username
   # adminPassword: password
+  # dbUser: username
+  # dbPassword: password
```

Comment:
If would prefer something like this, i. e. everything in one place in the config. This also allows multiple roles.

```yaml
auth:
  additionalUser:
    username: myuser
    password: mypass
    roles:
      - role: role1
        database: mydb
      - role: role2
        database: mydb
```

Reply:
@unguiculus would you say it is more common to have one user/pass for multiple roles / databases or multiple users with one role / db per user?  My instinct tells me the latter versus the former and would prefer to do things this way if I'm going to take the time to make this enhancement.  The latter could be something like this instead of your proposition:

```
auth:
  additionalUser:
    - username: myuser
      password: mypass
      role: role1
      db: mydb
    - username: myuser2
      password: mypass2
      role: role2
      db: mydb
```